### PR TITLE
Fixed Docker build for nodejs demo

### LINKED ISSUE #41: NPM Integration Dockerfie not working
The npm integration docker file here https://github.com/LNP-BP/rgb-sdk/blob/master/bindings/npm/Dockerfile is out of sync with the current sdk directory structure.

Managed to create a working docker file to run the nodejs demo in a container as below 

DockerFile:
```
FROM rust:1.47.0-slim-buster as builder

COPY bindings/npm /rgb-sdk/bindings/npm
COPY librgb /rgb-sdk/librgb

RUN apt-get update -y \
    && apt-get install -y \
        cmake \
        python3 \
        pkg-config \
        make \
        perl \
        g++ \
        git \
        wget \
        libssl-dev \
        libzmq3-dev \
        libpcre3-dev \
        libpq-dev

RUN wget https://freefr.dl.sourceforge.net/project/swig/swig/swig-4.0.1/swig-4.0.1.tar.gz && \
    tar xf swig-*.tar.gz && \
    cd swig-* && \
    ./configure && make -j12 && \
    make install

RUN wget -qO- https://raw.githubusercontent.com/nvm-sh/nvm/v0.35.3/install.sh | bash && \
   . $HOME/.nvm/nvm.sh && \
   nvm install v10 --lts && \
   nvm alias default v10

WORKDIR /rgb-sdk/bindings/npm

RUN . $HOME/.nvm/nvm.sh && npm install --unsafe-perm

FROM node:10-buster-slim

RUN apt-get update -y \
    && apt-get install -y \
        libzmq5 \
        libssl1.1 \
        libpq5 \
    && apt-get clean \
    && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*

RUN mkdir -p /rgb-sdk/librgb/target/release/ /rgb-sdk/bindings/npm/

COPY --from=builder /rgb-sdk/bindings/npm/build/Release/rgblib.node /rgb-sdk/bindings/npm/build/Release/
COPY --from=builder /rgb-sdk/librgb/target/release/librgb.so /rgb-sdk/librgb/target/release/
COPY demo/nodejs /rgb-sdk/demo/nodejs
COPY wrappers/ecmascript /rgb-sdk/wrappers/ecmascript
COPY bindings/npm/rgblib.js /rgb-sdk/bindings/npm/

WORKDIR /rgb-sdk
```

This can be built by
```
$ docker build -f bindings/npm/Dockerfile -t rgb-sdk-npm .

# Enter the container in interactive mode
$ docker run -it rgb-sdk-npm /bin/bash

# Run nodejs demo
$ cd demo/nodejs && node example.js


If this looks ok, will make a PR for both docker file and instruction document.

### PR DESCRIPTION
**Pull request by rajarshimaitra**
Fixes #41

### CODE REVIEW

File: bindings/npm/Dockerfile [hunk 1]
```diff
@@ -45,9 +43,12 @@ RUN apt-get update -y \
...
-RUN mkdir -p /librgb/target/release/ /librgb/bindings/nodejs/
+RUN mkdir -p /rgb-sdk/librgb/target/release/ /rgb-sdk/bindings/npm/
 
-COPY --from=builder /rgb-sdk/bindings/nodejs/build/Release/rgblib.node /librgb/
-COPY --from=builder /rgb-sdk/librgb/target/release/librgb.so /librgb/target/release/
+COPY --from=builder /rgb-sdk/bindings/npm/build/Release/rgblib.node /rgb-sdk/bindings/npm/build/Release/
+COPY --from=builder /rgb-sdk/librgb/target/release/librgb.so /rgb-sdk/librgb/target/release/
+COPY demo/nodejs /rgb-sdk/demo/nodejs
+COPY wrappers/ecmascript /rgb-sdk/wrappers/ecmascript
+COPY bindings/npm/rgblib.js /rgb-sdk/bindings/npm/
```
Comment (dr-orlovsky) [inline bindings/npm/Dockerfile]: Why do we need these?

File: bindings/npm/README.md [hunk 2]
```diff
@@ -17,18 +17,19 @@ cd bindings/npm
...
 
 ```bash
-docker build -f bindings/npmnpm/Dockerfile -t rgb-sdk-npm .
-docker run --rm -v $(pwd):/opt/mount --entrypoint bash \
-    rgb-sdk-npm \
-    -c 'mkdir -p /opt/mount/librgb/target/release /opt/mount/bindings/npm/build/Release \
-    && cp /rgb-sdk/target/release/librgb.so /opt/mount/librgb/target/release/librgb.so \
-    && cp /rgb-sdk/rgb.node /opt/mount/bindings/npm/build/Release/rgb.node'
-```
+$ docker build -f bindings/npm/Dockerfile -t rgb-sdk-npm .
```
Comment (dr-orlovsky) [inline bindings/npm/README.md]: This will make the thing unusable with anything else than running a sample demo right inside the docker container

**Review by dr-orlovsky — CHANGES_REQUESTED** [review]

File: bindings/npm/Dockerfile (hunk 1 above)
Comment (rajarshimaitra) [reply bindings/npm/Dockerfile]: These were required to run the demo.

File: bindings/npm/README.md (hunk 2 above)
Comment (rajarshimaitra) [reply bindings/npm/README.md]: I wasn't sure why a volume was added and copied the `rgb.node` and `librgb.so` to there. I modified the readme to add instruction to run the demo. Earlier it wasn't clear to me what the docker file is supposed to do. And in the readme instruction, we were just building it and copying those two files into a volume.

It's possible that I am missing the purpose of the docker build. I just made it to run the `example.js`. If it's not what it was ment for, do suggest and I will change accordingly.

Comment (dr-orlovsky) [reply bindings/npm/README.md]: The docker is used to build `rgb.node`, which is a compiled NodeJS module file, which must be used on the local machine. That is the purpose of the docker and it has nothing to do with the example

Comment (rajarshimaitra) [reply bindings/npm/README.md]: Ok so I will remove the demo part. Do you want those copying instructions into a volume to be kept?

Comment (dr-orlovsky) [reply bindings/npm/README.md]: Yes, they have to be kept. There was nothing to change in this file other than directory names; no lines should be added or removed

**Comment by rajarshimaitra** [comment]
Updated reverting unwanted changes.

**Review by dr-orlovsky — APPROVED** [review]

**Pull request merged**